commit b0fd976b97d626c8506c729496708a910cd2a863
Author: Jocelyn Le Sage <jocelyn@le-sage.com>
Date:   Sun Feb 26 22:17:17 2023 -0500

    Make sure to lowercase email address entered by the user during login.

diff --git a/backend/internal/token.js b/backend/internal/token.js
index a64b901..ed9a45f 100644
--- a/backend/internal/token.js
+++ b/backend/internal/token.js
@@ -24,7 +24,7 @@ module.exports = {
 
 		return userModel
 			.query()
-			.where('email', data.identity)
+			.where('email', data.identity.toLowerCase().trim())
 			.andWhere('is_deleted', 0)
 			.andWhere('is_disabled', 0)
 			.first()
